build.sbt

```diff
@@ -30,6 +30,7 @@ inThisBuild(Def.settings(
   scmInfo := Some(
     ScmInfo(url("https://github.com/apache/incubator-pekko-http"), "git@github.com:apache/incubator-pekko-http.git")),
   description := "Apache Pekko Http: Modern, fast, asynchronous, streaming-first HTTP server and client.",
+  resolvers += "Apache Pekko Staging".at("https://repository.apache.org/content/groups/staging"),
```

Comment:
seems like we will need this always during the RC progress?

Reply:
that is where we put the staging jar files for - the jars are only there during the staging process - when they are released or dropped, they are no longer in staging